indico/modules/events/abstracts/forms.py

```diff
@@ -168,15 +194,24 @@ def __init__(self, *args, **kwargs):
         if len(candidate_contrib_types) == 1:
             kwargs.setdefault('accepted_contrib_type', candidate_contrib_types[0])
         super(AbstractJudgmentForm, self).__init__(*args, **kwargs)
-        self.session.query = Session.query.with_parent(self.event).order_by(Session.title)
-        self.accepted_track.query = Track.query.with_parent(self.event).order_by(Track.title)
-        self.accepted_contrib_type.query = (ContributionType.query
-                                                            .with_parent(self.event)
-                                                            .order_by(ContributionType.name))
+
+
+class AbstractsJudgmentForm(AbstractJudgmentFormBase):
```

Comment:
I'll rename it to `BulkAbstractJudgmentForm` or leave it as it is, what do you think?